src/components/TransformWorkflow/FlowEditor/Components/OperationConfigLayout.tsx

```diff
@@ -82,6 +84,7 @@ const operationComponentMapping: any = {
     continueOperationChain,
     clearAndClosePanel,
     dummyNodeId,
+    action,
```

Comment:
Since all the props are same for all forms. Lets update it to this format.

const operationComponentMapping = {
  [RENAME_COLUMNS_OP]: RenameColumnOpForm,
  [JOIN_OP]: JoinOpForm
  ....
};

For getting the component you can do

const Form = operationComponentMapping["operation type"]

and use Form with the props.

Reply:
Agreed and makes sense. Done and pushed